app/src/main/res/values/strings.xml

```diff
@@ -235,7 +235,7 @@
         <item>Let A (B) denote the event that the first (second) card is an ace. These events are not independent, so you need to calculate Pr[A and B] = Pr[A]*Pr[B|A]. Pr[A] is the probability that your first card is an ace: that\'s 4/52. Pr[B|A] is the probability that your second card is an ace, given that the first was an ace (the vertical bar means \"given that\"). Can you figure out what that is? At that point, conditional on event A, you know there are 51 cards remaining, of which 3 are aces.</item>
         <item>Notice that there are two ways to get a single ace: either on the first card, or the second. Those are mutually exclusive events, so you have Pr[A or B] = Pr[A] + Pr[B], where A is the event that the first card is an ace and the second is not (and B is the reverse). Calculating Pr[A] is similar to the previous question: it\'s (4/40)*(36/39). What is Pr[B]?</item>
         <item>A little trick here is that, no matter what card you draw first, the probability that the second is of the same suit is&#8230;</item>
-        <item>Of the six possible preferences, only three are compatible with her first choice. For example, given what you know, there\'s zero probability that the monkey prefers blue to yellow to red, but there\'s a 1/3 probability that she prefers yellow to red to blue.</item>
+        <item>Of the six possible preferences, only three are compatible with her first choice. For example, given what you know, there\'s zero probability that the monkey prefers blue to yellow to red, but there\'s a 1/3 probability that she prefers yellow to red to blue. This puzzle was inspired by a New York Times article: see https://www.nytimes.com/2008/04/08/science/08tier.html and specifically the discussion following the sentence \"Dr. Chen says that the monkey’s distaste for blue can be completely explained with statistics alone\".</item>
```

Comment:
He has a bunch of articles about monkeys' choices at https://www.anderson.ucla.edu/faculty_pages/keith.chen/papers.htm.